lava-master/Dockerfile

```diff
@@ -89,6 +89,9 @@ COPY settings.conf /etc/lava-server/
 COPY device-types-patch/ /root/device-types-patch/
 RUN cd /etc/lava-server/dispatcher-config/device-types/ && for patch in $(ls /root/device-types-patch/*patch) ; do sed -i 's,lava_scheduler_app/tests/device-types/,,' $patch && patch < $patch ; done
 
+COPY lava-patch/ /root/lava-patch
+RUN cd /usr/lib/python3/dist-packages && for patch in $(ls /root/lava-patch/*patch) ; do patch -p1 < $patch ;done
```

Comment:
This should error out if any of the patches fail to apply.

Reply:
Fixed and fixed the same problem in device-types-patch